docs/conf.py

```diff
@@ -66,9 +66,7 @@
 # Inject custom css files in `/_static/css/*`
 html_static_path = ['_static']
 
-import sphinx_rtd_theme
 html_theme = "sphinx_rtd_theme"
-html_theme_path = [sphinx_rtd_theme.get_html_theme_path()]
```

Comment:
@ebezzi, do you remember why you added this?

Reply:
I believe that was a requirement for RTD.